contracts/modules/capital/SwapOperator.sol

```diff
@@ -458,7 +458,7 @@ contract SwapOperator {
 
       // avgAmountOut in ETH
       uint avgAmountOut = amountIn * netShareValue / (10 ** fromToken.decimals());
-      uint maxSlippageAmount = avgAmountOut * swapDetails.maxSlippageRatio / 1e18;
+      uint maxSlippageAmount = avgAmountOut * swapDetails.maxSlippageRatio / MAX_SLIPPAGE_DENOMINATOR;
```

Comment:
how come we are hitting this issue - isn't the slippage to withdraw 0 anyway?

Reply:
I agree the issue must be fixed, just trying to understand the behaviour

Reply:
The slippage bug is valid, however what you are describing here is not a consequence of the slippage (first of all because we don't have any slippage in fork tests unless we push a new price to chainlink), but rather a consequence of the enzyme withdrawal fees.

The provided fix is correct, but we should also fetch the withdrawal fee from enzyme and apply it on top of `avgAmountOut` and then applying the slippage on top of the result. With the withdrawal fee applied, the fork test should pass with a 0% slippage.

Reply:
Following the indications from Enzyme's team, i learned that to get the correct net share value we can basically trigger a small redemption (2 wei in this case) which triggers net share value recomputation.

I trigger that here:
https://github.com/NexusMutual/smart-contracts/pull/860/files#diff-eca555b76e500a76402b3318a3186118d210bf7a1157b2302b8b0e9e3b144ae4R462

That value is equal to gross share value after and they are both reflective of the fee applied (no fee application is needed.).


Once this happens i then execute the swap for the full amount.

Reply:
Following yesterday's chat discussion we settled for the oracle + slippage so I'm resolving this conversation.